### ISSUE TITLE
fatal error: cannot open file

### ISSUE THREAD

**Issue by masnagam** [masnagam/concc#5]
When building Chrome with concc, messages like below are shown frequently:

```
../../third_party/eigen3/src/Eigen/Core:278:10: fatal error: cannot open file '../../third_party/eigen3/src/Eigen/src/Core/AssignEvaluator.h': Operation not permitted
#include "src/Core/AssignEvaluator.h"
         ^
```

Maybe, sharing header files via SSHFS does not work properly.

**Comment by masnagam** [comment]
Confirmed that the same errors occur when running `autoninja` directly on the remote worker container without `cc_wrapper`.
This means that the root cause of the errors exits inside sshfs.

Also confirmed that the header files can be accessible from a shell on the remote worker container without any error.

**Comment by masnagam** [comment]
The same error occurs when building with root.

**Comment by masnagam** [comment]
Ran sshfs with `-d` and checked log messages, but I could not see `Operation not permitted` errors.

**Comment by masnagam** [comment]
Using atmoz/sftp or corilus/sftp as SSH server for sharing the workspace on the local PC seems to work fine.
Probably, my SSH server settings are wrong.

**Comment by masnagam** [comment]
fixed in 774056d3ff63c96095db57b26b1454610e355f91
